src/DocumentFormat.OpenXml/GeneratedCode/schemas_microsoft_com_office_spreadsheetml_2017_richdata2.g.cs

```diff
@@ -188,7 +188,7 @@ public StringValue RichSortKey
         }
 
         /// <summary>
-        /// <para>descending</para>
+        /// <para>descending, this property is only available in Office 2010 and later.</para>
```

Comment:
File name suggests that this is schema registered in 2017. Is it correct that property can be available in Office 2010? (multiple cases)

Reply:
the reason for this is that the containing complex type is defined in the Office 2019 schema http://schemas.microsoft.com/office/spreadsheetml/2017/richdata2 but the property this comment is describing and which is used by that containing type is actually one that has been around since Office 2010 in the namespace http://schemas.microsoft.com/office/spreadsheetml/2009/9/main.